requirements/dev.txt

```diff
@@ -4,14 +4,14 @@
 #
 #    make upgrade
 #
--e git+https://github.com/edx/xblock-sdk.git@c541e4fee8c90647b136f75b404e55c04c8b67f4#egg=xblock-sdk==0.1.7  # via -r requirements/test.txt
+-e git+https://github.com/edx/xblock-sdk.git@446e979bad1f3bf7f891787b32d5b2e7620b2f23#egg=xblock-sdk==0.1.9  # via -r requirements/test.txt
 appdirs==1.4.3            # via -r requirements/test.txt, -r requirements/travis.txt, fs, virtualenv
 astroid==2.3.3            # via -r requirements/test.txt, pylint, pylint-celery
 attrs==19.3.0             # via -r requirements/test.txt, pytest
 bleach==3.1.5             # via -r requirements/test.txt, readme-renderer
 click-log==0.3.2          # via -r requirements/test.txt, edx-lint
 click==7.1.2              # via -r requirements/pip-tools.txt, -r requirements/test.txt, click-log, edx-lint, pip-tools
-ddt==1.3.1                # via -r requirements/test.txt
+ddt==1.4.0                # via -r requirements/test.txt
```

Comment:
This is currently constrained in edx-platform, maybe we should do that in here:
https://github.com/edx/edx-platform/blob/e51bd9abd648c904b9c57651313190690dfab20f/requirements/constraints.txt#L16

Reply:
Actually, I am not sure about this, ddt is only used in the tests. So, I guess that this is fine as it is.